### ISSUE TITLE
Error downloading 7z package for iobituninstaller and  iobitdriverbooster

### ISSUE THREAD

**Issue by okibcn** [echoiron/echo-scoop#15]
So far updates for **iobituninstaller** and  **iobitdriverbooster** have been installed properly. But, today scoop is no longer able to install updates. I got this error:

```shell
$> scoop install iobitdriverbooster
Installing 'iobitdriverbooster' (10.0.0.32) [64bit] from echo bucket
The remote server returned an error: (403) FORBIDDEN.
URL https://xmod-my.sharepoint.cn/:u:/g/personal/admin_xmod_partner_onmschina_cn/ERhy9oRfwy9BmMPKakYnvqwBC49q_0HzL50yjqnsTeUpTA?download=1#/dl.7z is not valid
```

and 

```shell
$> scoop install iobituninstaller
Installing 'iobituninstaller' (12.0.0.9) [64bit] from echo bucket
The remote server returned an error: (403) FORBIDDEN.
URL https://xmod-my.sharepoint.cn/:u:/g/personal/admin_xmod_partner_onmschina_cn/EVL9E1BJa2VAkSRmNHz0jUABCnXt7_8DJJWf1PiwNci3sA?download=1#/dl.7z is not valid
```

The problem must be an incompatibility of this server with scoop, as I can download both 7z packages with Edge browser.

**Comment by echoiron** [comment]
![image](https://user-images.githubusercontent.com/52339367/193173473-9f8fae3c-90ec-4dd9-b450-e4c02f31ada2.png)
![image](https://user-images.githubusercontent.com/52339367/193173522-3c79414b-7ce0-4235-b9ec-41b3deb366bc.png)
Tested, the app installation is normal.

**Comment by okibcn** [comment]
I see you are using aria2 downloader. I tried the aria2 workaround and it works.

With the default scoop download process, it doesn't work. At least not from the US.

**Comment by echoiron** [comment]
scoop is not multi-threaded by default, and will break whenever there is a connection problem.
It is recommended to use aria2.
```bash
scoop instal aria2
```

**Comment by okibcn** [comment]
Yes, that is what I have done. It removed my problem. But I never had any problem in years downloading the package. This is the first time I had that problem and had to use aria2. I prefer keeping my system as lean as possible, this is why I try to install only the necessary packages.

In any case, the new server you are using is not compatible with Scoop out of the box. If you plan to continue using it, you probably should include a note in the readme file of this repo.
